### ISSUE TITLE
Where can we find Skribe?

### ISSUE THREAD

**Issue by lassik** [egallesio/STklos#554]
The pages at https://stklos.net/Doc/html/ say:

> This Html page has been produced by [Skribe](http://www.stklos.net/~eg/Publis/JFP05/article.html).

The link is broken. Where is the version of Skribe that is used to generate the HTML, and does it still work?

**Comment by egallesio** [comment]
Hi @lassik, 

Skribe was designed with Manuel Serrano. The link you see is to the first version of Skribe, written in Bigloo. Unfortunately, I don't think Manuel has been maintaining Skribe for a long time. The version you see at https://stklos.net/Doc/html/, was compiled with a version 2 of Skribe, written in STklos. This version was never released and was never really in a publishable state. Furthermore, it no longer works with versions of STklos higher than 1.20 (I had kept an image of this version just to produce the STklos documentation).  Since then, this documentation is in *asciidoctor*.

In short, I'm afraid I can't give you anything usable. 

Note to myself: it is really urgent to release version 2 of STklos, to have a website that does not talk about old stuff such as Skribe or ScmPkg anymore :wink:
